arm/Microsoft.Network/azureFirewalls/deploy.bicep

```diff
@@ -15,9 +15,6 @@ param azureSkuName string = 'AZFW_VNet'
 ])
 param azureSkuTier string = 'Standard'
 
-@description('Optional. Enable the preview feature for DNS proxy.')
-param enableDnsProxy bool = false
```

Comment:
Is this out of preview and hence removed?

Reply:
Yes. It can now be set via policy.